ftw/tabbedview/interfaces.py

```diff
@@ -40,6 +40,41 @@ class ITabbedView(Interface):
         )
 
 
+class ITabbedViewEndpoints(Interface):
+
+    def listing():
+        """Fetches the corresponding view which renders a listing.
+        Called from javascript tabbedview.js in reload_view line 58
```

Comment:
i would not reference the line here ... it will change for sure and then your pointer is confusing (and wrong 😢 )